libs/partners/openai/langchain_openai/chat_models/base.py

```diff
@@ -889,6 +916,35 @@ def _create_chat_result(
 
         return ChatResult(generations=generations, llm_output=llm_output)
 
+    def _create_chat_result_responses(
+        self, response: Response, generation_info: Optional[Dict] = None
+    ) -> ChatResult:
+        generations = []
+
+        if error := response.error:
+            raise ValueError(error)
```

Comment:
wonder if we can raise a more useful error here. the error object has a code and an error